src/lib.rs

```diff
@@ -19,10 +19,12 @@ use std::u32;
 use std::collections::HashMap;
 
 extern crate libc;
+extern crate combine;
 
 pub mod disassembler;
 pub mod ebpf;
 pub mod helpers;
+pub mod asm_parser;
```

Comment:
Ah also, can we keep includes in alphabetical order (in each one of the crate/public module/private module section)? Also, for dependencies in the `Cargo.toml` file?